components/atom/textarea/package.json

```diff
@@ -9,6 +9,7 @@
     "build:styles": "cpx './src/**/*.scss' ./lib"
   },
   "dependencies": {
+    "@s-ui/react-atom-input": "5",
```

Comment:
wont't there be css collision in those projects who have already this package installed but with an old major version?

Reply:
The only changes is the disabled addition for read-only prop

Reply:
#1289